src/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.CoreCLR.cs

```diff
@@ -193,6 +193,9 @@ private static void PrelinkCore(MethodInfo m)
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern int GetExceptionCode();
 
+        [MethodImpl(MethodImplOptions.InternalCall)]
+        internal static extern uint GetExceptionCount();        
```

Comment:
This does not belong here.

Could you please add to `System.Exception` instead?